### ISSUE TITLE
tmpmail on Homebrew!

### ISSUE THREAD

**Issue by gcpmusic** [sdushantha/tmpmail#55]
Would be great to update tmpmail via Homebrew.
Thanks

**Comment by chenrui333** [comment]
I can help with this issue.

@sdushantha if you can create a new release tarball file, which contains the binary executable, I can help shipping it out in the homebrew side. Thanks!

**Comment by vonProteus** [comment]
only thing that stops me from using it its not being available via brew

**Comment by sdushantha** [comment]
@chenrui333 You can find the tarball in the release section
https://github.com/sdushantha/tmpmail/releases

**Comment by vonProteus** [comment]
yes but it won't auto update :(

**Comment by legendofmiracles** [comment]
> @chenrui333 You can find the tarball in the release section
> https://github.com/sdushantha/tmpmail/releases

The releases are outdated, newest version according to the script is 1.1.9 but the release is 1.1.4

**Comment by sdushantha** [comment]
@legendofmiracles Sorry, I have made a new release now: https://github.com/sdushantha/tmpmail/releases/tag/v1.1.9

**Comment by piyush1104** [comment]
Did we ever release it to brew?

**Comment by lodenrogue** [comment]
Its a single bash file... I don't understand why you need homebrew

**Comment by vonProteus** [comment]
auto updates?
and ease of install

**Comment by gcpmusic** [comment]
Yes has been asked many time in the past years including me but never released on homebrew. I truly dont understand the complication. Almost anything is on homebrew so shouldn't be that complicated.

Thanks in advance if you finally decide to do so as well.

**Comment by lodenrogue** [comment]
Add the file to your PATH or put it in /usr/bin if you want every user on your machine to access it.

**Comment by lodenrogue** [comment]
Last commit was 2 years ago. I doubt this will get updates any time soon.

**Comment by chenrui333** [comment]
sorry for taking it too long, just filed a pr to include into brew.
